src/java.datatransfer/share/classes/java/awt/datatransfer/DataFlavor.java

```diff
@@ -1286,6 +1286,9 @@ public boolean isFlavorTextType() {
 
     /**
      * Serializes this {@code DataFlavor}.
```

Comment:
This most definitely changes the serialisation spec. So a CSR is needed.
Also shouldn't readExternal be updated to correspond ?

Reply:
There's not a great number of "good" examples of this in the JDK, so probably OK except it
seems like most cases will do it like a normal javadoc method so you'd want an @param tag too.

note : this is a desktop class, so I'm looking at this one but someone else will need to look at all the others

Reply:
I've created a CSR with a slightly modified version of the text proposed in this PR: 
https://bugs.openjdk.org/browse/JDK-8348408

Other doc tags such as `@param` and `@throws` are inherited from the overridden method. Also, this is the only change in this PR that modifies the serialization spec.

Reply:
CSR was approved, and I updated the PR with the slightly modified spec text from the CSR.